src/HTMLMesh.js

```diff
@@ -243,6 +257,38 @@ function html2canvas( element ) {
 
 	}
 
+	function getStyleForElement( element ) {
+		const style = window.getComputedStyle( element );
+		if ( element instanceof HTMLButtonElement ) {
+			if ( element.dataset.hoverStyleStored !== 'true' ) {
+				// The first time we render, store hover style in data attributes to make
+				// hover style work in VR because style aren't recomputed when we add the
+				// hover class.
+				if ( element.dataset.hoverStyleStored !== 'pending' ) {
+					// The mutation observer won't trigger rerender if we add the class now because we're currently rendering (scheduleUpdate is defined)
+					// Wait end of render before we modify the DOM.
+					element.dataset.hoverStyleStored = 'pending';
+					setTimeout( () => {
+						// Disable any css transition to avoir having a
+						// background color in between states when rerender is triggered.
+						element.style.transitionDuration = '0s';
+						element.classList.add( 'hover' );
+					}, 0 );
```

Comment:
Perhaps `queueMicrotask()` instead of `setTimeout(,0)`?

Reply:
That works too. Thanks.